docs/en/serial-vault.md

```diff
@@ -0,0 +1,415 @@
+
+Note that it takes some time to create the key. 
+
+After completion, verify creation with the following command:
+
+```bash
+snapcraft list-keys
+```
+This will return a list of the known keys. At the moment, the key you just added will be marked as 'not registered'.
+
+```bash
+    Name 	SHA3-384 fingerprint
+-   serial   yOkSks8x1OICgx60WW-YEKehTeWvQpE2qA17ZM-RT8i0u7YYrIOjSp1DARkv4Cuu 
+(not registered)
+```
+
+### Registering the key
+
+Register the key to the Brand SSO account as follows:
+
+```bash
+snapcraft register-key serial
+```
+
+You can confirm the key has been added with the command:
+
+```bash
+snapcraft list-keys
+```
+...which will now list the key with an asterisk (`*`) denoting that it is registered:
+
+```bash
+    Name 	SHA3-384 fingerprint
+*   serial   yOkSks8x1OICgx60WW-YEKehTeWvQpE2qA17ZM-RT8i0u7YYrIOjSp1DARkv4Cuu
+```
+
+### Exporting the key
+
+Before uploading the key, it must be encrypted. Export the key using ASCII
+armored encryption, as follows:
+ 
+```bash
+gpg --homedir ~/.snap/gnupg --armor --export-secret-key serial > serial.asc
+```
+
+In this example, serial.asc is the serial key file to upload to Serial Vault.
+You name a key used to sign system-user assertions differently.
+
+### Uploading the key to the Serial Vault
+
+1. Login to the Serial Vault as the Brand SSO account.
+1. Select Models.
+
+
+
+![image][serial-vault-image2]
+1. Click the Signing Keys + button.
+
+
+
+![image][serial-vault-image3]
+
+
+
+![image][serial-vault-image4]
+
+1. Enter account-id of the Brand SSO account as the Signing Authority.
+1. Under Signing Key, click Browse, then find and select the previously
+created ASCII-armored serial key file.. 
+1. Click Save. 
+The list of models and signing keys displays, including this new one.
+
+## Creating a model
+
+You need to create a model. This corresponds to the model previously used when
+building an image. (Every system has a model.) At runtime, the system sends
+the model to the Serial Vault, which uses it to authenticate the system before
+generating a serial assertion.
+
+The model contains key data that must exactly match data in the model in the
+deployed system, including:
+
+ - brand-id: this is the account-id of the Brand SSO account
+ - model name
+
+Also, the system itself must have a gadget snap whose prepare-device hook
+defines required snapd variables. See “Gadget Snap” section below for details
+on these variables. 
+
+Create a model as follows:
+
+1. Click Models.
+On first use, you have no models.
+1. Click the “+” button to the right of Models:
+
+
+![image][serial-vault-image5]
+
+
+![image][serial-vault-image6]
+
+1. Enter the account-id of your Brand SSO account into the Brand field.
+1. Enter the model name (from your system’s model assertion) into the Model
+field.
+1. The API Key must match the setting in your gadget snap prepare-device hook
+(see Gadget Snap section above). If you leave this field empty, an API key is
```

Comment:
Are we able to link to section when we refer to them instead of just simply mentioning a previous or later section?